src/pages/docs/step-ca/provisioners.mdx

```diff
@@ -820,18 +820,31 @@ for more guidance on configuring and using the ACME protocol with `step-ca`.
 
 ### SCEP
 
-The SCEP provisioner can sign and renew certificates using the SCEP protocol ([RFC8894](https://datatracker.ietf.org/doc/html/rfc8894)). SCEP is very popular for use in network equipment and mobile device management (MDM). It runs over HTTP using POSTed binary data or base64-encoded GET parameters, using CMS (PKCS#7) and CSR (PKCS#10) data formats. A shared secret authenticates clients to the CA.
+The SCEP provisioner can sign and renew certificates using the SCEP protocol ([RFC8894](https://datatracker.ietf.org/doc/html/rfc8894)). 
+SCEP is very popular for use in network equipment and mobile device management (MDM). 
+It runs over HTTP using POSTed binary data or base64-encoded GET parameters, 
+using CMS (PKCS#7) and CSR (PKCS#10) data formats. 
+A shared secret authenticates clients to the CA.
 
 #### Requirements
 
 Your CA must use an RSA intermediate CA, even if your client supports ECDSA.
-We use the RSA intermediate to encrypt/decrypt data, and you cannot do that with an ECDSA key.
+The RSA intermediate is used to encrypt/decrypt data, and you cannot do that with an ECDSA key.
```

Comment:
could we elaborate with a couple words what "data" is being encrypted and decrypted with the RSA key?

Reply:
Added the `pkcsPKIEnvelope` SCEP construct and the response data in b5ad71b12ef1327f1caaab8e4030554e5489fded.